instrumentation/opentelemetry_phoenix/lib/opentelemetry_phoenix.ex

```diff
@@ -22,11 +22,13 @@ defmodule OpentelemetryPhoenix do
       end
 
   """
-
-  require OpenTelemetry.Tracer
+  alias OpenTelemetry.SemanticConventions.Trace
```

Comment:
I think having it be `SemanticConventions.Trace` would make the attribute keys more clearly convey what they are, like `SemanticConventions.Trace.http_route() => meta.route()`.

`Trace` and `Tracer` are just so similar :). 

But I'm not hardset on it, so would be fine with merging as is as well.